C/tests/TestParser_1000_1999.cpp

```diff
@@ -2633,10 +2633,74 @@ void TestParser::case1967() {}
 void TestParser::case1968() {}
 void TestParser::case1969() {}
 
-void TestParser::case1970() {}
-void TestParser::case1971() {}
-void TestParser::case1972() {}
-void TestParser::case1973() {}
+void TestParser::case1970()
```

Comment:
For the parser tests I like to separate each token by a space, e.g. `__builtin_choose_expr ( 0 , x , y )` and `__builtin_choose_expr ( 1 , sizeof ( int ) , sizeof ( double ) )`.

This helps  me focusing on syntax purely, without associating semantics to it — given that ambiguity is important for psyche-c. It also makes grep consistent "everywhere" (which also is the reason why I follow the convention of `x` for the first identifier, `y` for the second, etc).